{"name": "test_case[case6]", "status": "broken", "statusDetails": {"message": "TypeError: 'tuple' object is not callable", "trace": "self = <test_case.Test_Case object at 0x000001DA927EDCC0>\ncase = {'case_description': '获取网站信息', 'case_expect': '{\"title\":\"V2EX\"}', 'case_method': 'get', 'case_params': '', ...}\n\n    @pytest.mark.parametrize('case',Excel_Handler().get_excel_data())\n    def test_case(self,case):\n>       response = RequestHandler(case).get_response()\nE       TypeError: 'tuple' object is not callable\n\nscripts\\test_case.py:13: TypeError"}, "parameters": [{"name": "case", "value": "{'case_project': 'v2EX项目', 'case_description': '获取网站信息', 'case_url': 'https://www.v2ex.com/api/site/info.json', 'case_method': 'get', 'case_params': '', 'case_expect': '{\"title\":\"V2EX\"}', 'other': ''}"}], "start": 1574774735900, "stop": 1574774736634, "uuid": "da6c0383-0a77-423d-bbdb-9ef763a2c21b", "historyId": "9e3029f99b52b4b13487f22dcc2f7b85", "testCaseId": "89a27520489a549f0caeaffa7948244d", "fullName": "scripts.test_case.Test_Case#test_case", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "Test_Case"}, {"name": "host", "value": "IG6I7Y1F5I7TZH5"}, {"name": "thread", "value": "14484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}